login-fire-common-behavior.html

```diff
@@ -319,6 +330,8 @@
         console.debug('The element ' + this.is + (this.noSignOut ? ' cannot' : ' can') + ' sign out.');
       }
       if (!this.noSignOut && this.signedIn) {
+        // Set _inProgress flag to true when a signout is started
+        this._inProgress = true;
```

Comment:
Set back the `_inProgress` property to `false` is not required here. It will be done inside the `_fireEvent` callback where it's a better place to do so.

Reply:
I'm confused about this comment? Is it maybe misplaced? _inProgress is set to true here. It will be set back once the success or error event is fired.

Reply:
What I am talking about here... 😒 . I check this part again.

Reply:
Please @Protoss78 , forgive me. It was 4AM in France when I wrote this shitty comment. 💩

Reply:
No problem at all. :+1: